Failed to compile master with clang 6.0.0.

I tried to build master (c9af205db89377b02d4f319a6e54fa9640aabeee) with clang 6.0.0, but it fails to compile with the message below. It builds fine with gcc, so I don't understand if this is a bug in deal or in clang.

```
/home/simon/workspace/dealiiCloseToMaster/include/deal.II/grid/tria_accessor.h:732:3: error: exception specification of explicitly defaulted move constructor does not   
      match the calculated one                                                                                                                                           
  TriaAccessor(TriaAccessor &&) noexcept = default;                                                                                                                      
  ^                                                                                                                                                                      
/home/simon/workspace/dealiiCloseToMaster/include/deal.II/grid/tria_accessor.h:2826:29: note: in instantiation of template class 'dealii::TriaAccessor<1, 1, 1>'         
      requested here                                                                                                                                                     
class CellAccessor : public TriaAccessor<dim, dim, spacedim>                                                                                                             
                            ^                                                                                                                                            
/home/simon/workspace/dealiiCloseToMaster/include/deal.II/grid/tria_accessor.h:3953:1: note: in instantiation of template class 'dealii::CellAccessor<1, 1>' requested   
      here                                                                                                                                                               
CellAccessor<1, 1>::point_inside(const Point<1> &) const;                                                                                                                
^                                                                                                                                                                        
/home/simon/workspace/dealiiCloseToMaster/include/deal.II/grid/tria_accessor.h:772:3: error: exception specification of explicitly defaulted move assignment operator    
      does not match the calculated one                                                                                                                                  
  operator=(TriaAccessor &&) noexcept = default;  
```

--- a/include/deal.II/grid/tria_accessor.h
+++ b/include/deal.II/grid/tria_accessor.h
@@ -729,7 +729,7 @@ public:
   /**
    * Move constructor.
    */
-  TriaAccessor(TriaAccessor &&) noexcept = default;
+  TriaAccessor(TriaAccessor &&) = default; // NOLINT
 
   /**
    * Conversion constructor. This constructor exists to make certain
@@ -769,7 +769,7 @@ public:
    * Move assignment operator. Moving is allowed.
    */
   TriaAccessor &
-  operator=(TriaAccessor &&) noexcept = default;
+  operator=(TriaAccessor &&) = default; // NOLINT
 
   /**
    * Defaulted destructor.

